exercises/concept/logs-logs-logs/.meta/design.md

```diff
@@ -20,3 +20,21 @@ This exercise's prerequisites Concepts are:
 - `strings`
 - `switch-statement`
 - `constructors`
+
+## Analyzer
+
+This exercise could benefit from the following rules in the [analyzer][analyzer]:
+
+1. Parse log level
+    - `LogLine.getLogLevel()`
+       - `actionable`: Verify that a switch statement was used. If they used if-else, comment that switch is better due to there being many cases.
```

Comment:
This is an interesting one - `switch` statements don't always appear in `LogLine.getLogLevel`. There are solutions that add a method to the `LogLevel` enum, like [this one](https://exercism.org/tracks/java/exercises/logs-logs-logs/solutions/Delereq).